core/src/main/java/nu/yona/server/analysis/service/ActivityDTO.java

```diff
@@ -18,22 +16,25 @@
 @JsonRootName("activity")
 public class ActivityDTO
 {
-	private Date startTime;
-	private Date endTime;
+	private ZonedDateTime startTime;
+	private ZonedDateTime endTime;
 
 	@JsonCreator
-	public ActivityDTO(@JsonProperty("startTime") Date startTime, @JsonProperty("endTime") Date endTime)
+	public ActivityDTO(@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ") @JsonProperty("startTime") ZonedDateTime startTime,
```

Comment:
You may create a new ticket for that to make merging this PR easier.

On Thu, May 12, 2016 at 7:47 AM, Bert Roos <notifications@github.com> wrote:

> In core/src/main/java/nu/yona/server/analysis/service/ActivityDTO.java
> <https://github.com/yonadev/yona-server/pull/173#discussion_r62967623>:
>
> >
> >  	@JsonCreator
> > -	public ActivityDTO(@JsonProperty("startTime") Date startTime, @JsonProperty("endTime") Date endTime)
> > +	public ActivityDTO(@JsonFormat(pattern = "yyyy-MM-dd'T'HH:mm:ss.SSSZ") @JsonProperty("startTime") ZonedDateTime startTime,
>
> Absolutely. I remembered an issue in using constants in annotations, so I
> didn't try it. Triggered by you, I tried it and it simply works.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/yonadev/yona-server/pull/173/files/e4dc2e653b6cb687e84ef9343b03564e5e25f97b#r62967623>
>